main.css

```diff
@@ -0,0 +1,582 @@
+  font-family: 'Source Sans Pro', sans-serif;
+  font-size: 14px;
+  color: #333;
+  line-height: 1.5;
+}
+
+figcaption>span {
+  font-family: 'Slabo 27px', serif;
+  color: #888888;
+
+}
+
+.author {
+  width: 50%;
+  float: left;
+
+}
+
+.author h1 {
+  font-size: 1.3em;
+  margin-bottom: 5px;
+  font-weight: 600;
+}
+
+.author p {
+  font-family: Verdana, Geneva, Tahoma, sans-serif;
+  font-size: 0.8em;
+
+}
+
+.article-social-media {
+  width: 50%;
+  float: left;
+  margin-top: 30px;
+}
+
+.article-social-media div {
+  float: right;
+
+}
+
+.article-social-media ul {
+  display: flex;
+}
+
+.article-social-media li {
+  padding: 10px 10px;
+}
+
+.header-clear {
+  clear: both;
+}
+
+.header-article {
+  margin-bottom: 20px;
+}
+
+figure>img {
+  width: 920px;
+
+}
+
+.article-content>figure {
+  display: block;
+  position: relative;
+  left: -100px;
+}
+
+.article-content>p {
+  line-height: 1.5;
+  margin-bottom: 15px;
+  font-size: 19px;
+  font-family: 'Hind Madurai', sans-serif;
+}
+
+.resize-section {
+  overflow: auto;
+  resize: vertical;
+}
+
+#space-fig {
+  margin-bottom: 360px;
+}
+
+#space-authors {
+  margin-bottom: 420px;
+
+}
+
+#space-cosmos {
+  margin-bottom: 80px;
+}
+
+#resize-img {
+  width: 720px;
+  position: absolute;
+  ;
+  top: 0;
+  left: 0;
+  z-index: 1;
+}
+
+#resize-img-container {
+  width: 720px;
+  left: 2px;
+  height: 360px;
+  top: 10px;
+  bottom: 300px;
+}
+
+#resize-figure {
+  width: 720px;
+  position: absolute;
+  height: 480pc;
+  margin: 15px 0 20px;
+
+}
+
+#resize-figure h2 {
+  font-size: 18px;
+  font-family: 'Noto Sans HK', sans-serif;
+  margin-bottom: 8px;
+}
+
+#resize-figure p {
+  color: #666666;
+  font-family: 'EB Garamond', serif;
+  font-size: 14px;
+}
+
+#resize-figure figcaption {
+  margin-top: 370px;
+}
+
+.figcaption-separator {
+  width: 50px;
+  height: 1px;
+  background: #333;
+  margin: 20px 0;
+  bottom: 1px;
+
+}
+
+#figcap {
+  display: block;
+  top: 500px;
+}
+
+.image-space {
+  height: 100px;
+}
+
+.text-container {
+  position: relative;
+  width: 180px;
+  float: left;
+  z-index: 2;
+  top: 140px;
+  font-size: 14px;
+  color: #2f2f2f;
+  line-height: 1.2;
+
+}
+
+.text-container strong {
+  font-weight: 600;
+}
+
+.img-floating {
+  position: relative;
+  margin-top: 10px;
+}
+
+.text-separator {
+  height: 150px;
+  position: relative;
+  clear: both;
+}
+
+.video-container {
+  margin: 40px 0;
+}
+
+.video-container div>p:first-child {
+  font-size: 1em;
+}
+
+.video-container p {
+  font-size: 0.8em;
+  color: #666;
+}
+
+#spirals-image {
+  position: static;
+  margin: 30px 0;
+}
+
+#spirals-image img {
+  width: 600px;
+}
+
+footer a {
+  text-decoration: none;
+  color: #999
+}
+
+footer p {
+  color: #999;
+}
+
+
+
+#cosmos-area {
+  display: grid;
+  grid-template-columns: 80% 20%;
+  width: 1100px;
+  margin: 0 auto;
+}
+
+#cosmos-area div {
+  padding: 4px;
+
+}
+
+
+#first-column-cosmos {
+  display: grid;
+  grid-template-columns: 33% 33% 33%;
```

Comment:
[OPTIONAL]
Since both the first and the second column cosmos are in the same row they should be defined in the grid template. This explains why there is an offset in your page. You can give 75% to the whole `More space in cosmos` and  25% for the `Most popular`